### DISCUSSION TITLE
Additional Categories

### DISCUSSION THREAD

**Discussion by orrious** [tagyoureit/nodejs-poolController#504] [Ideas]
Hardware: 
A place to document known good hardware.  Enclosures, relays (dry contact and solid state), transformers, breakers, temperature sensors, orp, ph, level, pressure, flow sensors, RS485 interfaces, gpio boards, etc.

NON-njs-pC:  (not married to the name, just the intent)
I would like a place to discuss NON-njs-pC topics where other folks can search for answers when they are designing their outdoor experience.  What works well, others experiences, etc.

Topics I would like to start with would be: 
Lighting
Audio
Pyrotechnics (electronic ignition systems)

Thoughts?

**Comment by tagyoureit** [comment]
This is a dumb technical limitation.  I have no problem adding more, but Github has a hard limit of [9 categories](https://github.com/github-community/community/discussions/7960) and we're already using all of them.  

Just pick the `General` category and feel to start any discussion you want.  If/when the limit is lifted we can add more official topics.
